gradle/versions.gradle

```diff
@@ -42,7 +42,7 @@ ext {
     robolectricVersion = "4.5.1"
     uiAutomatorVersion = "2.2.0"
     msal4jVersion = "1.10.0"
-    apacheHttpVersion = "4.5.13"
```

Comment:
msebraApacheHttpClientVersion?  You probably shouldn't arbritrarily switch these...

Reply:
Also, this change is causing build failures in your consumers.  Are there updates for them?

Reply:
yes - for MSAL and broker, I'm going to bundle them with the other PR.
For ADAL - I 'll open a separate one.